neurodamus/node.py

```diff
@@ -1301,8 +1301,7 @@ def run_all(self):
     # -
     @return_neuron_timings
     def _run_neuron(self):
-        Nd.load_hoc("ShowProgress")  # TODO: Drop this
-        _ = Nd.ShowProgress(Nd.cvode, MPI.rank)
+        SimulationProgress()
```

Comment:
IIRC we had to assign to something (hence `_ = ...`) so that the object is not immediately destroyed

Reply:
It seems to be working like that, maybe because of neuron, but I could do the _ = ... just to be sure